### ISSUE TITLE
Compiling error on example project

### ISSUE THREAD

**Issue by shannonchou** [iwasrobbed/Down#57]
Please help prevent duplicate issues before submitting a new one:

* [x] I've searched other open/closed issues for duplicates before opening up this new issue.

# Report

## What did you do?

Clone the repo and run example project.
* Git commit: 642c564
* Xcode: 9.0.1

## What did you expect to happen?

Running successfully.

## What happened instead?

Compiling error "Missing required module 'libcmark'"

![image](https://user-images.githubusercontent.com/3939980/32134745-595b2b3a-bbb9-11e7-93c5-688eae8a312f.png)

**Comment by iwasrobbed** [comment]
Thanks @shannonchou ; this should be fixed once this PR lands https://github.com/iwasrobbed/Down/pull/58
